src/README.md

```diff
@@ -32,16 +32,48 @@ var myApp = new springroll.Application({
     buttonSize: false, // whether or not the game has adjustable button sizes
     removableLayers: false, // whether or not the game supports the removal of distracting game layers
     hudPosition: false, // whether or not the game supports multiple HUD positions
-    difficulty: false, // whether or not the game supports an adjustable difficulty,
-    keyBinding: true, //whether or not the game supports re-mappable key bindings.
-    colorVision: true, //whether or not the game supports alternate color schemes for color blind users
+    hitAreaScale: true, // whether or not the game supports adjustable hit areas
+    dragThresholdScale: true, // whether or not the game supports adjustable drag thresholds
+    health: true, // whether or not the game supports adjustable health amounts
+    objectCount: true, // whether or not the game supports adjustable object count requirements
+    completionPercentage: true, // whether or not the game supports adjustable completion percentage requirements
+    speedScale: true, // whether or not the game supports adjustable speeds
+    timersScale: true, // whether or not the game supports adjustable timers
+    inputCount: true, // whether or not the game supports adjustable input count requirements
+    keyBinding: true, // whether or not the game supports re-mappable key bindings.
+    colorVision: true, // whether or not the game supports alternate color schemes for color blind users
   }
 });
 ```
 
 Note that if any of `vo`, `music`, or `sfx` are available features, `sound` will be marked as a feature implicitly.
 Also, all of these features are marked `false` by default.
 
+## Difficulty Features
```

Comment:
Let's add another little section with a general overview of what the numbers mean. For instance, we should note that all values are between 0 and 1. We also might want to add implementation details about what 0 and 1 maps to in the context of that feature.

Reply:
Added a brief description about what the difficulty values are and what they mean